charts/plex/1.6.16/ix_values.yaml

```diff
@@ -2,3 +2,7 @@ image:
   pullPolicy: IfNotPresent
   repository: plexinc/pms-docker
   tag: 1.25.2.5319-c43dc0277
+plexpass_image:
```

Comment:
shouldn't this be camelCase?

Reply:
Yes, that's a better practice with charts - i'll make the change. Thanks for pointing it out